### PR TITLE
add: bids + heudiconv lecture

### PR DESCRIPTION
**Pull request by mgxd**

### CODE REVIEW

File: 2017-03-boston/lectures/bids-heudiconv/presentation.md [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+    item: index within category
+    subject: participant id
+    seqitem: run number during scanning
+    subindex: sub index within group
+    """
+
+    data = create_key('run{item:03d}', outtype=('nii.gz',))
+    info = {data: []}
+    last_run = len(seqinfo)
+    for s in seqinfo:
```
Comment (satra) [inline 2017-03-boston/lectures/bids-heudiconv/presentation.md:169]: isn't `s` a namedtuple now?

File: 2017-03-boston/lectures/bids-heudiconv/presentation.md [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+  - separate tracking of specific metadata when processing dataset
+  - is my dataset missing anything?
+---
+## [Brain Imaging Data Structure](http://bids.neuroimaging.io) (or BIDS)
+
+<img src="assets/data2bids.jpg" width="100%" />
+.right[*Gorgolewski, K. J. et al. 2016*]
+
+
+???
```
Comment (satra) [inline 2017-03-boston/lectures/bids-heudiconv/presentation.md:71]: ???

File: 2017-03-boston/lectures/bids-heudiconv/presentation.md [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+### mathiasg@mit.edu
+
+---
+name: content
+layout: false
+
+## Roadmap
+
+.middle[
+.center[
```
Comment (satra) [inline 2017-03-boston/lectures/bids-heudiconv/presentation.md]: use `class: middle center` at the beginning of the slide section

File: 2017-03-boston/lectures/bids-heudiconv/presentation.md [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+### Is it BIDS yet?
+
+Let's check:
+- [In-browser validator](http://incf.github.io/bids-validator) / Command line
+
+--
+
+- A change to `heudiconv` will be coming soon that fixes most of these problems
+
+ - However, event files and participants.tsv will still need to be created
```
Comment (satra) [inline 2017-03-boston/lectures/bids-heudiconv/presentation.md]: walk through these bits as well.

File: 2017-03-boston/lectures/bids-heudiconv/presentation.md [hunk 5]
```diff
@@ -0,0 +1,413 @@
...
+name: extrasteps
+
+### Is it BIDS yet?
+
+Let's check:
+- [In-browser validator](http://incf.github.io/bids-validator) / Command line
+
+--
+
+- A change to `heudiconv` will be coming soon that fixes most of these problems
```
Comment (satra) [inline 2017-03-boston/lectures/bids-heudiconv/presentation.md:399]: what problems and changes? this should happen before tuesday then.

File: 2017-03-boston/lectures/bids-heudiconv/presentation.md (hunk 2 above)
Comment (mgxd) [reply 2017-03-boston/lectures/bids-heudiconv/presentation.md:71]: presentation mode notes

Comment (satra) [reply 2017-03-boston/lectures/bids-heudiconv/presentation.md:71]: aha - forgot about that.

File: 2017-03-boston/lectures/bids-heudiconv/presentation.md (hunk 1 above)
Comment (mgxd) [reply 2017-03-boston/lectures/bids-heudiconv/presentation.md:169]: yes? although that was implemented already before my changes

Comment (satra) [reply 2017-03-boston/lectures/bids-heudiconv/presentation.md:169]: we should really move towards using that.

File: 2017-03-boston/lectures/bids-heudiconv/presentation.md (hunk 5 above)
Comment (mgxd) [reply 2017-03-boston/lectures/bids-heudiconv/presentation.md:399]: To add:
* remove non-alphanumeric characters from subject name
* `sub-` should be added to output subject directory
* `TaskName` key to functional json

Eventually:
* **Smart** `IntendedFor` key for each fieldmap json 

[rough implementation](https://github.com/mgxd/makebids/blob/master/makebids/makebids.py)

Comment (satra) [reply 2017-03-boston/lectures/bids-heudiconv/presentation.md:399]: let's get these in. perhaps add a bids postprocessing function that gets called after conversion. whenever bids output is enabled. (in fact we should really consider refactoring this in the longer run with post-processors)

**Comment by satra** [comment]
perhaps replace the git annex get with datalad:

http://datasets.datalad.org/?dir=/dicoms/dartmouth-phantoms/PHANTOM1_3

**Comment by mgxd** [comment]
@satra this is good to go

**Pull request merged**